uaclient/cli.py

```diff
@@ -97,6 +98,38 @@ def new_f(args, cfg):
     return new_f
 
 
+def daemon_file_exists():
+    return os.path.exists(defaults.DEFAULT_UPGRADE_CONTRACT_FLAG_FILE)
+
+
+def get_running_services():
```

Comment:
let's move this into util module and maybe just define a function that does this work for us instead of jumping into a function call for a one-liner

is_service_running(service_name):
    running_services, _ = util.subp(["ps", "-aux"])
    return bool(service_name in running_services)